public/app/features/explore/Logs/LogsTableNavColumn.tsx

```diff
@@ -51,96 +53,51 @@ function getStyles(theme: GrafanaTheme2) {
   };
 }
 
+const collator = new Intl.Collator('en', { sensitivity: 'base' });
```

Comment:
Very nice.

Reply:
Should we use `en` in here? Other places in Grafana use `undefined`. So I am wondering if we should be consistent or if there is a specific reason to use this. 

<img width="831" alt="image" src="https://github.com/grafana/grafana/assets/30407135/9a903912-fe3c-45f9-984f-b9db0335f875">